handlers/aws/sqs_trigger.py

```diff
@@ -92,10 +101,10 @@ def _handle_sqs_event(config: Config, event: dict[str, Any]) -> Iterator[tuple[d
                     span.__exit__(None, None, None)
                     span = None
 
-                es_event = _default_event.copy()
+                es_event = deepcopy(_default_event)
```

Comment:
`.copy()` is shallow, `_default_event` contains `fields` dict that would be referenced and updated on every iteration, ending with all the events collected by the shipper to have the partially the same content